### PR TITLE
Feat/countdown

### PR DESCRIPTION
**Pull request by Thiraput01**
### Type of Change

- [* ] New features
- [ ] Bug fixes
- [ ] Breaking changes
- [ ] Refactor
- [ ] Documentation updates
- [ ] Other (please specify):

## Description
- feat: homepage
- feat: countdown timer

### CODE REVIEW

File: src/components/timercard/index.tsx
Comment (ChanatpakornS) [inline src/components/timercard/index.tsx:1]: using `cn`

File: src/app/home/page.tsx
Comment (ChanatpakornS) [inline src/app/home/page.tsx:1]: change `Link` tag position

File: src/components/timercard/index.tsx
Comment (Thiraput01) [reply src/components/timercard/index.tsx:1]: done

**Comment by ChanatpakornS** [comment]
Don't forget to change the `tape_home.svg` in public folder to naja

File: src/components/footer/index.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import { Icon } from '@iconify/react';
+import Link from 'next/link';
+
+export default function Footer() {
+    return (
+        <div className="flex w-full flex-row items-center justify-center gap-2.5 rounded-tl-lg rounded-tr-lg bg-project-yellow px-[104px] py-4 font-sov text-xl text-black">
+            <Link
+                href="https://www.instagram.com/larngearfest2024/"
+                className="flex w-full flex-row items-center justify-center gap-2.5"
+            >
```
Comment (TeeGoood) [inline src/components/footer/index.tsx]: เปลี่ยน w-full เป็น mx-auto

**Review by TeeGoood — APPROVED** [review]

**Review by TeeGoood — APPROVED** [review]
LGTM!!!

File: package-lock.json
Comment (punchanabu) [inline package-lock.json:1]: I dont think we need `package-lock.json` since we use `pnpm` kub

**Review by punchanabu — APPROVED** [review]

**Pull request merged**